src/one_dragon/base/push/channel/one_bot.py

```
@@ -176,4 +180,143 @@ def validate_config(self, config: dict[str, str]) -> tuple[bool, str]:
+        url: str,
+        data: dict,
+        headers: dict[str, str],
+        label: str,
+    ) -> tuple[bool, str]:
+        """发送单批合并转发请求"""
+        try:
+            resp = requests.post(url, data=json.dumps(data), headers=headers, timeout=30)
+            resp.raise_for_status()
+            result = resp.json()
+            if result.get('status') == 'ok':
+                log.info(f'OneBot {label}成功！')
```

Ruff (0.15.10)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)